data-serving/scripts/converters.py

```diff
@@ -120,37 +137,43 @@ def convert_age(id: str, age: str) -> Dict[str, Any]:
                 age_range['end'] < -1 or age_range['end'] > 300):
             raise ValueError('age_range.end outside of valid range')
 
-        return {
-            'ageRange': age_range
-        }
+        return age_range
     except ValueError:
         logging.warning('[%s] [demographics.age] value error %s', id, age)
 
 
+def convert_sex(id: str, sex: str) -> str:
+    if pd.isna(sex):
+        return None
+
+    try:
+        if str(sex).lower() not in VALID_SEXES:
+            raise ValueError('sex not in enum')
+
+        return str(sex).capitalize()
+    except ValueError:
+        logging.warning('[%s] [demographics.sex] value error %s', id, age)
+
+
 def convert_demographics(id: str, age: str, sex: str) -> Dict[str, Any]:
     '''Converts age and sex columns to the new demographics object. '''
     demographics = {}
 
     converted_age = convert_age(id, age)
     if converted_age:
-        demographics['age'] = converted_age
-
-    if pd.notna(sex):
```

Comment:
Was there an issue with this? Or just sneaking a little enhancement into the PR?

Reply:
yeah just a sneaky little cleanup, can't control myself